db/db_test_util.cc

```diff
@@ -449,12 +449,15 @@ Options DBTestBase::GetOptions(
       break;
     }
     case kBlockBasedTableWithPartitionedIndexFormat3: {
-      table_options.index_type = BlockBasedTableOptions::kTwoLevelIndexSearch;
-      options.prefix_extractor.reset(NewNoopTransform());
+      table_options.format_version = 3;
       // Format 3 changes the binary index format. Since partitioned index is a
       // super-set of simple indexes, we are also using kTwoLevelIndexSearch to
       // test this format.
-      table_options.format_version = 3;
+      table_options.index_type = BlockBasedTableOptions::kTwoLevelIndexSearch;
+      options.prefix_extractor.reset(NewNoopTransform());
```

Comment:
Sorry I missed it previously but why do we want to set prefix extractor?

Reply:
I had copy-pasted it from kBlockBasedTableWithPartitionedIndex. Removing it.